### ISSUE TITLE
Support RandomizedSearchCV

### ISSUE THREAD

**Issue by vigsterkr** [ratschlab/mlschema-model-converters#2]

**Comment by chrisbarber** [comment]
`RandomizedSearchCV` takes argument `param_distributions` which can accept a `scipy.stats.distributions` instance. When calling `sklearn_model.get_params()` this comes back as an `rv_frozen` instance. This is recapitulated as the name of the distribution and the args that were passed to it, eg
```python
uniform(loc=0, scale=4)
```
becomes
```json
{
    "dist_name": "uniform",
    "args": [],
    "kwds": {
        "loc": 0,
        "scale": 4
    }
}
```

**Comment by vigsterkr** [comment]
could we do this via PR, from now on as i have some comments that needs to be address that i'll put it directly to the commits now... but that'd make easier the dev cycle... hence i've created a kanban that is review based

**Comment by chrisbarber** [comment]
yes sorry, forgot about the kanban. just let me know the procedure and i will follow from now on
